### PR TITLE
 test(log4js mock): Prevent leaking state 

### PR DESCRIPTION
**Pull request by nicojs**

### CODE REVIEW

File: packages/stryker/test/unit/initializer/StrykerInitializerSpec.ts [hunk 1]
```diff
@@ -69,6 +73,7 @@ describe('StrykerInitializer', () => {
     });
 
     it('should prompt for test runner, test framework, mutator, transpilers and reporters', async () => {
+      console.error('RUNNING should prompt for test runner, test framework, mutator, transpilers and reporters');
```
Comment (simondel) [inline packages/stryker/test/unit/initializer/StrykerInitializerSpec.ts]: Why did you add this statement?

**Comment by nicojs** [comment]
Because i was debugging and it was late. Good catch!

**Comment by simondel** [comment]
Awesome thanks!
